src/app/modules/frontend/frontend.controller.ts

```diff
@@ -116,15 +115,38 @@ export const generateRedirectUrl: ControllerHandler<
   }
 }
 
+// Duplicated here since the feature manager is being deprecated.
+// TODO (#2147): delete this.
+enum FeatureNames {
+  Captcha = 'captcha',
+  GoogleAnalytics = 'google-analytics',
+  Sentry = 'sentry',
+  Sms = 'sms',
+  SpcpMyInfo = 'spcp-myinfo',
+  VerifiedFields = 'verified-fields',
+  WebhookVerifiedContent = 'webhook-verified-content',
+}
+
 /**
  * Handler for GET /frontend/features endpoint.
  * @param _req - Express request object
  * @param res - Express response object
  * @returns Current featureManager states
+ * @deprecated as the feature manager has been deprecated. This endpoint
+ * now hardcodes the feature states to support old clients.
+ * TODO (#2147): delete this
  */
 export const showFeaturesStates: ControllerHandler<
   unknown,
-  typeof featureManager.states
+  Record<FeatureNames, boolean>
 > = (_req, res) => {
-  return res.json(featureManager.states)
+  return res.json({
+    captcha: true,
```

Comment:
Should we reference the enum here?

```ts
return res.json({
  [FeatureNames.Captcha]: true,
  [FeatureNames.GoogleAnalytics]: true,
  ...
})
```

Reply:
sure can also! the keys are restricted by the controller type but okay